src/callbacks_stage/positivity_zhang_shu.jl

```diff
@@ -38,6 +38,13 @@ function (limiter!::PositivityPreservingLimiterZhangShu)(u_ode, integrator,
     end
 end
 
+# Version used by the AMR callback
+function (limiter!::PositivityPreservingLimiterZhangShu)(u, mesh, equations, solver,
```

Comment:
So for me to adapt the hot fix in TrixiSW I would need to add a similar function like
```julia
function (limiter!::PositivityPreservingLimiterShallowWater) blah blah blah
```
right?